canonical-web-and-design/vanilla-framework
# Add styling for table captions

## Done

Adds styling for caption element in table (for React MainTable).

Related to canonical-web-and-design/react-components#155


## QA

- Open [demo](https://vanilla-framework-3675.demos.haus/docs/examples/base/table-empty)
- Check if table captions renders correctly
- Review updated documentation:
  - https://vanilla-framework-3675.demos.haus/docs/base/tables#empty

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="1251" alt="Screenshot 2021-03-31 at 15 10 39" src="https://user-images.githubusercontent.com/83575/113149459-514e3780-9233-11eb-93bd-fccec6d03680.png">
